### DISCUSSION TITLE
cockroachdb support

### DISCUSSION THREAD

**Discussion by dominiquegarmier** [RobertCraigie/prisma-client-py#380] [Q&A] [answered]
Hello. I was wondering if this supports cockroachdb?

**Comment by RobertCraigie** [comment, marked as answer]
Yes, CockroachDB is supported, see https://www.prisma.io/docs/concepts/database-connectors/cockroachdb for reference.

I have not tested CockroachDB with the Python client however all connection handling / querying is done internally by Prisma so nothing should have to change for us to support it.

Also some PostgreSQL specific features will not be available when using CockroachDB with the Python client yet.

**Reply by dominiquegarmier** [reply]
ok intresting. I assume query raw will work just fine in those edge cases?

**Reply by RobertCraigie** [reply]
Yeah it will, if I remember correctly the only PostgreSQL specific feature that is enabled specifically in this client is case insensitive filtering.

Also any docstring that provides an SQL query example will use the wrong parameter format.

Apart from the cases mentioned above everything else should work exactly the same.
